locations/philmel.yml

```diff
@@ -0,0 +1,198 @@
+---
+location: philmel
+location_nice: Philipp-Melanchthon-Kirche
+latitude: 52.465881
+longitude: 13.434112
+contact_nickname: 'roedel'
+contacts:
+  - '@roedel:matrix.org'
+
+hosts:
+
+  - hostname: philmel-core
+    role: corerouter
+    model: "linksys_e8450-ubi"
+    wireless_profile: freifunk_default
+
+  - hostname: philmel-nf-east-5ghz
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    wireless_profile: freifunk_default
+    mac_override:
+      eth0: cc:2d:e0:9c:4f:00
+
+  - hostname: philmel-nf-s-5ghz
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    wireless_profile: freifunk_default
+    mac_override:
+      eth0: cc:2d:e0:9c:4f:00
+
+snmp_devices:
+
+  - hostname: philmel-rhnk
+    address: 10.230.2.4
+    snmp_profile: af60
+
+  - hostname: philmel-nord-5ghz
+    address: 10.230.2.6
+    snmp_profile: airos_8
+
+  - hostname: philmel-ost
+    address: 10.230.2.7
+    snmp_profile: airos_6
+
+  - hostname: philmel-vaterhaus
+    address: 10.230.2.8
+    snmp_profile: airos_8
+
+  - hostname: philmel-sued
+    address: 10.230.2.9
+    snmp_profile: airos_6
+
+  - hostname: philmel-sued-5ac
+    address: 10.230.2.10
+    snmp_profile: airos_8
+
+  - hostname: philmel-west
+    address: 10.230.2.11
+    snmp_profile: airos_6
+
+  - hostname: philmel-ak36
+    address: 10.230.2.12
+    snmp_profile: airos_8
+
+# TODO: Add all devices and
+# change naming
+airos_dfs_reset:
+  - name: "philmel-ak36"
+    target: "10.230.2.12"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+  - name: "philmel-nord-5ac"
+    target: "10.230.2.6"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+
+ipv6_prefix: "2001:bf7:820:1500::/56"
+
+# Complete subnet: 10.230.2.0/24
+# MGMT Network: 10.230.2.0/28
+# Mesh Network: 10.230.2.16/28 (-23)
+# DHCP Network: 10.230.2.32/28
+
+networks:
+  - vid: 2
+    role: dhcp
+    prefix: 10.230.2.32/28
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: true
+    enforce_client_isolation: true
+    assignments:
+      philmel-core: 1
+  # northeast mesh 5GHz ac
+  - vid: 10
+    role: mesh
+    name: mesh_no_5ghz         # Peers: kiehlufer-core
+    prefix: 10.230.2.17/32
+    ipv6_subprefix: -1
+    mesh_metric: 1024
+  # northwest mesh 5GHz ac
+  - vid: 11
+    role: mesh
+    name: mesh_nw_5ghz          # Peers: liegewiese, emser97
+    prefix: 10.230.2.18/32
+    ipv6_subprefix: -2
+    mesh_metric: 1024
+  - vid: 12
+    role: mesh
+    name: mesh_ost            # Peers: Area51, delbrueck66
+    prefix: 10.230.2.19/32
+    ipv6_subprefix: -3
+    mesh_metric: 1024
+  - vid: 13
+    role: mesh
+    name: mesh_nf_east
```

Comment:
On other locations we use the following naming convention here: `mesh_11s_[direction]_[band]` e.g. `mesh_11s_o5`